HOK.MissionControl/HOK.MissionControl.Core/Schemas/Addins.cs

```diff
@@ -88,13 +89,13 @@ private static string GetOffice()
 
                 // (Konrad) If we got a Source string ie. Machine Name we can proceed
                 // to extract the user location/office from it using regex pattern.
-                var match = Regex.Match(sourceString, settings.Pattern, RegexOptions.IgnoreCase);
-                if (match.Success && match.Groups.Count >= settings.Group)
-                {
-                    return match.Groups[settings.Group].Value;
-                }
+                var matches = Regex.Matches(sourceString, settings.Pattern, RegexOptions.IgnoreCase);
+                if (matches.Count <= settings.Match) return string.Empty;
```

Comment:
If `settings.Match` defaults to and uses 1-based numbering in Mission Control, this should just be `if (matches.Count < settings.Match)...`. Or we should adjust it to be 0-based at some point, either when saving to the DB from the web, or when pulling from the DB on the .NET side.

 Obviously, Mission Control could use 0 as the default for the match/group indices instead of dealing with any of the above.

Reply:
Yeah, I realized that too. I think we would want to unify how these matches are displayed in a tool like: https://regex101.com/ where everything is 1 indexed. I will log that under issues to pick up later when I am doing documentation so we can point people to the online tool, and keep it consistent with what we expect them to input here.